### PR TITLE
Set up React and add first page

### PR DESCRIPTION
**Pull request by Samad198**
Added the react app and documentation on how to get it working with the main app.
Should work by just running npm install from the react app then running the docker app as normal

### CODE REVIEW

File: react-client-app/src/stories/Introduction.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (spco) [inline react-client-app/src/stories/Introduction.stories.mdx:1]: Can we delete this file? And any others?

**Review by spco — COMMENTED** [review]
Looks generally good. The main feedback is getting it to run via Docker. We might need to put our heads together to get that working, but in general it will be that you need to do all your install in `Dockerfile` (and `Dockerfile.deploy` once it's working) and then startup instructions go into `entrypoint.sh`.

File: react-client-app/src/components/Test.jsx [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+import { Formik, Field, Form, ErrorMessage, FieldArray as FormikActions } from 'formik'
+import { getConceptLoop }  from '../api/values'
+import ConceptTag from './ConceptTag'
+import ToastAlert from './ToastAlert'
+import axios from 'axios'
+
+
+const api = axios.create({
+    baseURL: `https://609e52b633eed8001795841d.mockapi.io/`
```
[the remark this answers is not in the document]
Comment (Samad198) [reply react-client-app/src/components/Test.jsx:51]: That was a mock api that Joe was using at first. The whole of Test and Test2 components can be deleted now

**Comment by spco** [comment]
As it stands, this PR can be merged. It needs the addition of `COCONNECT_DB_AUTH_TOKEN` and `CCOM_APP_URL` to the `.env` files and Azure settings.

In future, a separate PR will handle the mounting of the react directory as a volume mount. For now, any changes in the react-client-app directory will require a rebuild.

**Pull request merged**